client/src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Message } from '../types';
+
+interface Props {
+  id: string;
```

Comment:
We're not using this, so I don't think we should include it in the interface or the props

Reply:
Are you talking about "id: string"?

Reply:
Yes